linter/orb.yaml

```diff
@@ -16,8 +16,21 @@ commands:
         description: |
           Optional alternate config file.
         type: string
+      pre_commit_version:
```

Comment:
nit: probably just `version`, given this is scoped as `pre-commit/pre_commit_version` right now and can't refer to anything else.